helm/templates/admission-webhooks/operator-webhook-with-cert-manager.yaml

```diff
@@ -0,0 +1,222 @@
+{{- if and (.Values.admissionWebhooks.create) (.Values.admissionWebhooks.certManager.enabled) }}
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: {{ printf "%s/%s-serving-cert" .Release.Namespace (include "amazon-cloudwatch-observability.name" .) }}
```

Comment:
is this -serving-cert correct cert name ?

Reply:
Yeah this mainly needs to match the name of the certificate itself [here](https://github.com/aws/amazon-cloudwatch-agent-operator/pull/29/files#diff-43bd35744a58a4155b7a13ec827ab3a8660fa7d8e1154123f11a508e01505c5bR11)